src/features/promise-combinators.md

```diff
@@ -20,7 +20,7 @@ Here’s an overview of the four combinators:
 :::table-wrapper
 | name                                        | description                                     | status                                                          |
 | ------------------------------------------- | ----------------------------------------------- | --------------------------------------------------------------- |
-| [`Promise.allSettled`](#promise.allsettled) | does not short-circuit                          | [proposal](https://github.com/tc39/proposal-promise-allSettled) |
```

Comment:
This won't officially happen until ES2020 is finalized in January. I'd like to continue linking to the proposal at least until then.

Currently, the spec PR for `allSettled` is not even merged yet :)